<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1248</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_linha">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Linha:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_coluna">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Coluna:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_linha1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_coluna">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_matriz">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Matriz:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_linha2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>linha2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_coluna2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>coluna2:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_linha2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>400</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_coluna2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>440</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_matriz2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>320</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Matriz:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_triangula_superior2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Triangular Superior</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Tinf">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Triangular Inferior</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_simetrica">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Simétrica</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_identidade">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Identidade</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_soma">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_subtrai">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>280</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_multiplica">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_divide">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>310</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_permuta">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>180</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Permutação</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ortogonal">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Ortogonal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>280</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Iguraldade</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>310</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Diferença</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_resultado_2">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>150</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Resultado:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_matriz">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>311</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_matriz_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>350</y>
      <width>311</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_matriz_3">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>180</y>
      <width>311</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Cria1">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Criar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Cria2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>420</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Cria 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_permuta_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>520</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Permutação</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ortogonal_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Ortogonal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_identidade_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>520</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Identidade</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_simetrica_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Simétrica</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_triangula_superior2_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Triangular Superior</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Tinf_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>490</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Triangular Inferior</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Transposta</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>550</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Transposta</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1248</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
